<?xml version="1.0"?>
<!-- URDF model to show single servo motor -->
<robot name="motor">
  <joint type="revolute" name="joint_tilt">
    <origin xyz="0 0 0" rpy="0 0 0"/>
    <child link="horn_tilt">horn_tilt</child>
    <parent link="base_link">base_link</parent>
    <axis xyz="0 0 1"/>
    <limit effort="100" velocity="100.0" lower="-500" upper="500" />
  </joint> 
<joint type="revolute" name="joint_pan">
    <origin xyz="0 0 0" rpy="0 0 0"/>
    <child link="horn_pan">horn_pan</child>
    <parent link="horn_tilt">horn_tilt</parent>
    <axis xyz="0 0 1"/>
    <limit effort="100" velocity="100.0" lower="-500" upper="500" />
  </joint> 
  <link name="base_link">
    <visual>
      <origin xyz="0 0 0" rpy="0 0 0"/>
      <geometry>
        <mesh filename="package://kondo_driver/urdf/meshes/KRS4000.dae" scale="1 1 1" />
      </geometry>
    </visual>
  </link>
  <link name="horn_tilt">
    <visual>
      <origin xyz="0 0 0.005" rpy="3.1415 0 0"/>
      <geometry>
        <mesh filename="package://kondo_driver/urdf/meshes/HORN.dae" scale="1 1 1" />
      </geometry>
    </visual>
  </link>
  <link name="horn_pan">
    <visual>
      <origin xyz="0 0 0.005" rpy="3.1415 0 0"/>
      <geometry>
        <mesh filename="package://kondo_driver/urdf/meshes/HORN.dae" scale="1 1 1" />
      </geometry>
    </visual>
  </link>
</robot>
